scripts/legacy/reorganize_model_directory.py

```
@@ -16,33 +16,33 @@
     # Create models directory if it doesn't exist
     if not os.path.exists(models_dir):
         os.makedirs(models_dir)
         print(f"✅ Created models directory: {models_dir}")
-    
+
     # 1. Save current model as model_1 (fallback)
     print(f"\n💾 SAVING CURRENT MODEL AS FALLBACK")
     print("-" * 40)
-    
+
     if os.path.exists(current_model_path):
```

Ruff (0.12.2)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)